rails_event_store_active_record/lib/rails_event_store_active_record/mappers/yaml.rb

```diff
@@ -0,0 +1,31 @@
+module RailsEventStoreActiveRecord
+  module Mappers
+    class YAML
+      def initialize(serializer: ::YAML)
+        @serializer = serializer
+      end
+
+      def create_event(event)
+        Event.create!(
```

Comment:
create!/persistance can be extracted here, back to repository (to return hash)... but this will reduce flexibility